papers/whitepaper.md

```diff
@@ -0,0 +1,54 @@
+
+The Atoma Network will be empowered by its TOMA token. Through a robust tokenomics, both compute providers, protocol participants and developers will be rewarded by their contribution to maintaining the network longevity. The TOMA token will be required to pay for AI inference fees, for nodes to deposit collateral to be able to participate on the network, and also for developers to being paid for their contributions to the Atoma Network. Moreover, the tokenomics follow an initial period of inflation in order to subsidize the network's growth. This growth includes both incentivizing node operators to acquire better compute hardware, to incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions, etc), as well as to incentivize developers to build applications that drive demand for the available compute resources on the Atoma Network. After the network's inflationary period (which will last a few years), the tokenomics will follow a more balanced dynamics, through tail emissions and burning mechanisms. Together, both mechanisms will lead to a gradual decrease in the total supply of the TOMA token, contributing for its increase in value provided the demand for Atoma Network increases, while also allowing for a some new allocation of the token to the protocol's treasury and to value providers (in the form of new node providers, model creators, developers, etc).
+
+In order to empower the next generation applications in the intersection of Web3 and AI, both data and model privacy is of utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user. This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party. Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data economic revenue models, as well applications that can operate on the user's behalf without revealing anything about the user. 
+
+On another note, the development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
+
```

Comment:
The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.